GDJS/Runtime/websocket-debugger-client/websocket-debugger-client.js

```diff
@@ -1,14 +1,64 @@
+/**
+ * An client side implementation of the Debugger
+ * @interface
+ */
+gdjs.DebuggerClient = function() {} // Necessary for Interface definition 
+
+/**
+ * Update a value of the {@link RuntimeGame} instance.
+ * @function
+ * @name gdjs.DebuggerClient#set
+ * @param {?} path - The path to modify.
+ * @param {any} newValue - The new Value.
+ */
+
+/**
+ * Call a method of the {@link RuntimeGame} instance.
+ * @function
+ * @name gdjs.DebuggerClient#call
+ * @param {?} path - The path to the method.
+ * @param {Array<any>} args - The arguments to pass the method.
+ */
+
+/**
+ * Dump all the relevant data from the {@link RuntimeGame} instance and send it to the Server.
+ * @function
+ * @name gdjs.DebuggerClient#sendRuntimeGameDump
+ */
+
+/**
```

Comment:
No capitals to Profiling. Be careful not to capitalize words in the middle of your sentences.